spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -80,20 +88,20 @@ private int incrementCounterAndReturnCurrentValue(TemporaryMetricsAggregation tm
         return (int) counter.count();
     }
 
-    private String createCounterIdFromMetric(TemporaryMetricsAggregation tma) {
+    private String createCounterIdFromMetric(TemporaryMetricsAggregation tma, ZonedDateTime cetTimeNow) {
         StringJoiner stringJoiner = new StringJoiner(SEPARATOR);
         stringJoiner.add(PREFIX)
                 .add(tma.getServiceApiName())
                 .add(tma.getRequestMethod())
                 .add(tma.getApiKeyValue())
                 .add(tma.getMetricType().toString())
                 .add(String.valueOf(tma.getHttpResponseCode()))
-                .add(LocalDate.now().toString())
+                .add(java.sql.Date.valueOf(cetTimeNow.toLocalDate()).toString())
```

Comment:
The `java.sql.Date.valueOf()` seems to be not required.

Reply:
Correct. I removed it.